setup.py

```diff
@@ -15,7 +15,7 @@
 
 setup(
     name="gyp-next",
-    version="0.8.0",
+    version = "0.8.1",
```

Comment:
```suggestion
    version="0.8.1",
```

Reply:
psf/black formatting.

Reply:
I opened https://github.com/nodejs/gyp-next/pull/96 to update the release action. Maybe it's fixed in a recent version. If it's not, I'll open an issue there.

Reply:
> I opened #96 to update the release action. Maybe it's fixed in a recent version. If it's not, I'll open an issue there.

also cc @bcoe

Reply:
Reported at https://github.com/googleapis/release-please/issues/777